create_references_for_ntjoin.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+
+# Split a fasta file into multiple files, one per sequence and create a reference config file for ntjoin
+# Usage: split_fasta.py <fasta file> <output directory> <config file>
+
+import sys
+
+input = sys.argv[1]
+output = sys.argv[2]
+config = sys.argv[3]
+ref_list = []
+with open(input, 'r') as f:
+    lines = f.readlines()
+    for i in range(len(lines)):
+        if lines[i].startswith('>'):
+            header = lines[i].strip()
+            filename = header.split(' ')[0].replace('>', '') + '.fasta'
```

Comment:
Just use `strip(">")`

Reply:
The headers usually have space between texts, so If we were to use `strip(">")`, the filename would end up being something like `NC.1234.1 species name complete mitochondrion.fasta` which is not ideal. `filename = header.split('  ')[0].replace('>', '') + '.fasta'`, on the other hand, creates `NC.1234.1.fasta`.

Reply:
But you are already splitting by a space and selecting the first in the list?

Reply:
what about` filename = header.strip('>').split(' ')[0] + '.fasta'`

Reply:
Yup, either that or `filename = header.split(' ')[0].strip('>') + '.fasta'` (that's what I meant initially)

Reply:
Done